### ISSUE TITLE
This plugin's menu item doesn't appear in WP 5.6x

### ISSUE THREAD

**Issue by mnelson4** [WordPress/wp-jquery-update-test#4]
Maybe this is temporary, but I'm finding on WP version 5.6-alpha-48686 that this plugin's menu item under "Plugins" doesn't appear. 

https://i.imgur.com/qzyV4RB.png

On WP 5.5 is appears fine.

**Comment by reykjalin** [comment]
I don't see the menu item in WordPress 5.6-RC1-49694 either.

**Comment by reykjalin** [comment]
When I try to open `wp-admin/plugins.php?page=wp-jquery-update-test` (presumably the URL to the settings page judging by the `save_settings` function:

https://github.com/WordPress/wp-jquery-update-test/blob/trunk/class_wp_jquery_update_test.php#L152-L153

I'm informed that

>Sorry, you are not allowed to access this page.

<details><summary>Screenshot</summary>

![image](https://user-images.githubusercontent.com/13835680/100433418-0631e580-30d6-11eb-900d-b8c66c448007.png)

</details>

This seems weird since I'm the admin on the site, and the only user to have ever existed on the site, and there are no file permission issues on the site.

**Comment by mnelson4** [comment]
Hey @reykjalin I since uncovered that this plugin was only intended to work on WP 5.5... see https://plugins.trac.wordpress.org/browser/wp-jquery-update-test/trunk/wp-jquery-update-test.php#L23
But it doesn’t say that on the GitHub read me, or the plugin readme, or a FAQ. I think it should mention “only works with WP 5.5” in all those places.

**Comment by reykjalin** [comment]
@mnelson4 yeah you're right! I reached out to @azaozz regarding this issue and he informed me that:

1. Latest release only works for WP < 5.6; and
1. The latest `trunk` should work with WP 5.6. Just download the repo zip and install that.

Barring that, an updated release is coming soon™️ 🙂

**Comment by azaozz** [comment]
Right, this was intentional. Older versions of this plugin are not intended to work on newer versions of WordPress (no point).
